assets/javascripts/initializers/admin-plugin-configuration-nav.js

```diff
@@ -0,0 +1,30 @@
+import { PLUGIN_CONFIG_NAV_MODE_TOP } from "discourse/lib/admin-plugin-config-nav";
+import { withPluginApi } from "discourse/lib/plugin-api";
+
+export default {
+  name: "discourse-ai-admin-plugin-configuration-nav",
+
+  initialize(container) {
+    const currentUser = container.lookup("service:current-user");
+    if (!currentUser || !currentUser.admin) {
```

Comment:
Makes me wonder if we should have a name for the inverse of admin, e.g. `currentUser.plebeian`. 😛